cli/clistat/container.go

```diff
@@ -0,0 +1,61 @@
+package clistat
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+
+	"github.com/spf13/afero"
```

Comment:
Curious about the choice to use afero vs. std [fs](https://pkg.go.dev/io/fs#FS)?

Reply:
Mainly for `afero.MemMapFS`, which is convenient for testing.